streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -307,11 +309,27 @@ void handleLostAll() {
         }
     }
 
+    /**
+     * @return Map from task id to that task's overall lag across all state stores
+     */
+    public Map<TaskId, Integer> getTaskLags() {
+        final Map<TaskId, Integer> taskLags = new HashMap<>();
+
+        for (final TaskId id : tasksOnLocalStorage()) {
+            if (isActive(id)) {
+                taskLags.put(id, ACTIVE_TASK_SENTINEL_LAG);
+            } else {
+                taskLags.put(id, 0);
```

Comment:
```suggestion
                taskLags.put(id, STANDBY_TASK_SENTINEL_LAG);
```

Either that, or my preference would actually be to inline both sentinel lags (with a comment explaining why that choice of sentinels).